graphql-server/Makefile

```diff
@@ -10,6 +10,7 @@ test:
 .PHONY: build
 build:
 	go build -o graphql-serverd cmd/graphql-server/main.go
+	go build -o migratord cmd/migration/main.go
```

Comment:
migrator is not daemon tho?

Reply:
it's used by the k8s job, but i guess we can `go run` it

Reply:
i mean we can still compile it, but maybe just name it migrator